mpls-over-udp-ip/draft-ietf-detnet-mpls-over-udp-ip.xml

```diff
@@ -152,6 +152,7 @@
  <t>
   These requirements are satisfied by the DetNet over MPLS Encapsulation
   described in <xref target="I-D.ietf-detnet-mpls"/> and they are partly satisfied
+  (requirement 1 and 4) 

```

Comment:
perhaps:
, i.e., IP flows cane be identified while no DetNet sequence number is carried ,